eduiddashboard/views/permissions.py

```diff
@@ -32,26 +33,23 @@ class PermissionsView(BaseFormView):
 
     schema = Permissions()
     route = 'permissions'
-    attributte = 'eduPersonEntitlement'
+    attribute = 'eduPersonEntitlement'
 
     def get_template_context(self):
         tempcontext = super(PermissionsView, self).get_template_context()
         ma = self.context.main_attribute
-        user = get_session_user(self.request, raise_on_not_logged_in = False, legacy_user = True)
-        if not user:
-            user = OldUser({})
-        if self.context.user.get(ma) == user.get(ma):
+        user = get_session_user(self.request, raise_on_not_logged_in = False)
+        if self.context.user.get(ma) == user.to_dict().get(ma):
```

Comment:
this main_attribute thing should die. just use eppn, unless that would break anything.